package.json

```diff
@@ -1,25 +1,25 @@
 {
   "name": "ethereumjs-vm",
-  "version": "2.2.2",
+  "version": "2.3.0",
   "description": "an ethereum VM implementation",
   "main": "index.js",
   "dependencies": {
     "async": "^2.1.2",
     "async-eventemitter": "^0.2.2",
-    "ethereum-common": "https://github.com/ethereumjs/common",
+    "ethereum-common": "0.2.0",
```

Comment:
Won't this break in the mean-time that the 0.2.0 is tagged on github, but not pushed to npm?

We can use git urls like `git+https://github.com/ethereumjs/common.git#v0.2.0` instead.

Reply:
I'm in the process of amending these release PRs to do exactly this.

Reply:
Note: comment is outdated, this has been discussed on Gitter, leave npm versions, PRs just have to be merged/published on npm in given order.